package.json

```diff
@@ -297,6 +297,7 @@
     "aws4": "^1.11.0",
```

Comment:
All I see are audit vulnerabilities lol

Reply:
can you elaborate

Reply:
@AdamAdHocTeam is there an actionable item here?